portfolio/src/main/webapp/script.js

```diff
@@ -59,6 +59,44 @@ async function deleteComments() {
     }
 }
 
+/** a map of the countries with the highest happiness score */
+class HappinessChart {
+  constructor(divID) {
+    google.charts.load('current', {
+      'packages':['geochart'],
+      // Note: you will need to get a mapsApiKey for your project.
+      // See: https://developers.google.com/chart/interactive/docs/basic_load_libs#load-settings
+      'mapsApiKey': 'AIzaSyCQ2pBJaaQ0JIitodS6-sbHfie2Qosefxg'
```

Comment:
[suggestion] Might want to keep this private. In a way that you push the files without it